packages/@react-spectrum/breadcrumbs/src/Breadcrumbs.tsx

```diff
@@ -56,8 +56,14 @@ function Breadcrumbs<T>(props: SpectrumBreadcrumbsProps<T>, ref: DOMRef) {
 
   let updateOverflow = useCallback(() => {
     let computeVisibleItems = (visibleItems: number) => {
-      let listItems = Array.from(listRef.current.children) as HTMLLIElement[];
-      let containerWidth = listRef.current.offsetWidth;
+      // N.B. refs can be null at runtime.
```

Comment:
odd question, but what is N. B.?

Reply:
_Nota bene_, meaning "observe carefully" :nerd_face:

Reply:
FWIW, I think once TS is flagging this sort of access, this would not be worthy of a callout. The compiler would make it self-evident.

Happy to rephrase or remove the comment if the larger concern of nulls/undefined is already well known among the maintainers here.

Reply:
I'm not worried about calling it out, but maybe since we have a fairly diverse set of people who need to be able to read the code, we can just leave off the N B, the fact that there is a comment is probably enough to say "warning here"